torch/csrc/jit/passes/onnx/eliminate_unused_items.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <torch/csrc/jit/ir/ir.h>
+
+namespace torch {
+namespace jit {
+
+void EliminateUnusedItemsONNX(
```

Comment:
Could you add comments to explain the major purpose of this pass. I am not clear, why dce and lint won't cover this pass without your explanation.

Reply:
Sure, I added  the comment explaining the pass